fcl/gen/numi/prodcorsika_genie_protononly_icarus_numi_fullosc_volDetEnclosure_tpc.fcl

```diff
@@ -0,0 +1,12 @@
+#include "prodcorsika_genie_protononly_icarus_numi_volDetEnclosure_tpc.fcl"
```

Comment:
I suggest to add a comment to describe in words what "full oscillation" means, like:
```suggestion
#
# Purpose: NuMI neutrino interactions (after "full" oscillation) and cosmic ray generation in ICARUS.
#
# Full oscillation: all muon neutrinos and antineutrinos are turned into electron ones, and vice versa.
# Generation is limited to an envelope volume of all TPC active volumes (plus a small margin).
#

#include "prodcorsika_genie_protononly_icarus_numi_volDetEnclosure_tpc.fcl"
```
And you can also add a `# Author: Joseph Zennamo (jaz8600@fnal.gov)` to the mix, for blaming purposes.

Reply:
Done!